MetaCell/salk-interactive-atlas
# Feature/63

Closes #63 

Implemented solution: 
Replaces mocked experiment data with call to the api.


How to test this PR: 
1. Create an experiment
2. Add a population to the experiment
3. Open experiment page

Expected:
- See the populations on the left sidebar

4. Active the population

Expected:
- See the cells showing up with the color of the population

![image](https://user-images.githubusercontent.com/19196034/159766501-55f8b2ab-ee1a-4f5b-a3a4-e277c13d2854.png)


### Sanity checks:

### Breaking changes (select one):

### Possible deployment updates issues (select one):

### Test coverage (select one):

### Documentation (select one):

### Nice to have (if relevant):
